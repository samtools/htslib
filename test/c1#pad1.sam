@SQ	SN:c1	LN:10
s0a	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s0b	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s0c	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s1	0	c1	1	0	5M6I5M	*	0	0	AACCGGTTAACCGGTT	*
s2	0	c1	1	0	5M1P4I1P5M	*	0	0	AACCGTTAACGGTT	*
s3	0	c1	1	0	5M3I3P5M	*	0	0	AACCGGTTCGGTT	*
s4	0	c1	1	0	5M3P3I5M	*	0	0	AACCGAACCGGTT	*
s5	0	c1	1	0	4M1D2P2I2P1D4M	*	0	0	AACCTAGGTT	*
s6	0	c1	1	0	2M3D6I3D2M	*	0	0	AAGTTAACTT	*
