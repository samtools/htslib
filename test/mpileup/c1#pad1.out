c1	1	9	^!A^!A^!A^!A^!A^!A^!A^!A^!A	~~~~~~~~~
c1	2	9	AAAAAAAAA-3()	~~~~~~~~~
c1	3	9	CCCCCCCC*	~~~~~~~~~
c1	4	9	CCCCCCCC-1()*	~~~~~~~~~
c1	5	9	GGGG+6(GTTAAC)G+6(*TTAA*)G+6(GTT***)G+6(***AAC)*+6(**TA**)-1()*+6(GTTAAC)-3()	~~~~~~~~~
c1	6	9	CCCCCCC**	~~~~~~~~~
c1	7	9	GGGGGGGG*	~~~~~~~~~
c1	8	9	GGGGGGGG*	~~~~~~~~~
c1	9	9	TTTTTTTTT	~~~~~~~~~
c1	10	9	T$T$T$T$T$T$T$T$T$	~~~~~~~~~
